utils/build-universal.sh

```diff
@@ -16,3 +16,17 @@ MKTGZ_SCRIPT=`dirname "$0"`/macosx-make-tgz.sh
 . $INST_SCRIPT
 
 $MKTGZ_SCRIPT "$R_LIBS/$pkgname"
+
+# Fail if there are links to the builder's gfortran
+so_path="$R_LIBS/$pkgname/libs"
+if [ -d "$so_path" ]; then
+    for arch in $TARGET_ARCHS; do
+        arch_so_path="${so_path}${arch}"
+        for so_file in $arch_so_path/*.so; do
+            bad_shared_libraries=`otool -L "$so_file" | grep /opt/gfortran`
+            if [ ! -z "$bad_shared_libraries" ]; then
+                exit 1
```

Comment:
Would it be useful to print a warning/error mentioning the problematic .so file before exiting here ?

Reply:
The errors are displayed at https://github.com/Bioconductor/BBS/commit/6e5bcaccdf873cd52be172f4d742e6520d55b7a4.